### PR TITLE
Added paragraph and heading styles to bookmark section

### LINKED ISSUE #39: Add CC vocabulary paragraph and heading styles to bookmarks section
Add appropriate classes of cc-vocabulary to heading and paragraph of the bookmarks section and also remove unnecessary style definitions from corresponding SCSS files.

**How to do it**
Note - [CC-vocabulary]() is already linked into the project, therefore you just need to add the appropriate classes(which you can see after inspecting the element in dev tools) and mimic the markup(if different) to change the styles.

**Paragraph**
- [Code in question](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/popup.html#L89)
- "No bookmarks yet" message is a negative indication, therefore use the corresponding classes from the [Paragraph section](https://opensource.creativecommons.org/cc-vocabulary/styleguide/#/Elements/Paragraph) of cc vocabulary docs.
- The current classes present in the element are necessary, so wrap the paragraph in a div with those classes.

**Heading**
- [Code in question](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/popup.html#L77)
- Add the classes and use h3 instead of h2.
- Remove the unnecessary styles from __bookmarks.scss_.
- To align the heading, wrap the heading in a div with class `text-align-center`

### PR DESCRIPTION
**Pull request by sharmaadityaHQ**
Fixes #39 

**Description**  
Added paragraph and heading styles to bookmark section.  

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

**Comment by makkoncept** [comment]
@sharmaaditya570191 please add the screenshots of the changes

**Comment by sharmaaditya570191** [comment]
Hi @makkoncept I will add the screenshot of the changes soon.

**Comment by sharmaaditya570191** [comment]
**Before**
Screenshot from 2019-12-04 20-12-35

**After**
Screenshot from 2019-12-04 20-24-36

**Comment by sharmaaditya570191** [comment]
Hi @makkoncept . I have made final changes and added screenshot. Please have a look.

File: src/popup/popup.html [hunk 1]
```diff
@@ -101,7 +103,7 @@ <h2>Bookmarks</h2>
       <div class="container">
         <div id="spinner-placeholder--bookmarks"></div>
         <section class="section-content section-content--bookmarks">
-          <div class="row">
+          <div class="row vocab paragraph inherit-colored negative-indicating">
```
Comment (makkoncept) [inline src/popup/popup.html]: Please add these classes to the paragraph element, not to the div.

File: src/popup/popup.html [hunk 2]
```diff
@@ -90,7 +90,9 @@
     <div id="snackbar-bookmarks" class="snackbar"></div>
     <div class="section-bookmarks">
       <div class="section-bookmarks--header">
-        <h2>Bookmarks</h2>
+        <div class="text-align-center vocab heading">
```
Comment (makkoncept) [inline src/popup/popup.html]: Add the class to the heading element, not to the div.
Also, as your screenshot shows, the current size of heading is too small, please adjust it accordingly.

**Review by makkoncept — CHANGES_REQUESTED** [review]

**Comment by sharmaaditya570191** [comment]
**After changes** 
Screenshot from 2019-12-06 19-06-42

**Comment by makkoncept** [comment]
The issue which this PR tries to solve also says to remove the redundant CSS styles from `_bookmarks.scss`. Please do that.

Remove the h2 styles. [see here](https://github.com/creativecommons/ccsearch-browser-extension/blob/master/src/popup/sass/components/_bookmarks.scss#L11)

**Comment by sharmaaditya570191** [comment]
Screenshot from 2019-12-07 10-14-04

**Comment by sharmaaditya570191** [comment]
Please have a look @makkoncept

**Pull request merged**